core/src/engine/engine.cc

```diff
@@ -919,7 +919,7 @@ namespace jiminy
         {
             return ::jiminy::saveToBinary(value);
         }
-    };
```

Comment:
Is this intended ? This seems unvalid...

Reply:
Should disappear after a rebase. It was a "bug" in jiminy master, after merging something a bit too fast.